src/components/RecipeDetails/RecipeDetails.js

```diff
@@ -1,76 +1,79 @@
 import PropTypes from 'prop-types';
 import { Table } from 'react-bootstrap';
-import React, { Component } from 'react';
+import React from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
-export default class RecipeDetails extends Component {
-  static propTypes = {
-    maxVG: PropTypes.bool.isRequired,
-    percentVG: PropTypes.number.isRequired,
-    shakeAndVape: PropTypes.bool.isRequired,
-    steepDays: PropTypes.number.isRequired,
-    flavors: PropTypes.arrayOf(PropTypes.object).isRequired
-  };
+export default function RecipeDetails({
+  maxVg,
+  percentVg,
+  shakeAndVape,
+  steepDays,
+  flavors
+}) {
+  const percentPg = 100 - percentVg;
+  const flavorTotal = flavors.reduce((accum, current) => {
+    return accum + current.percent;
+  }, 0);
 
-  static defaultProps = {
-    maxVG: false,
-    percentVG: 50,
-    shakeAndVape: false,
-    steepDays: 0,
-    flavors: []
-  };
-
-  render() {
-    const { maxVG, shakeAndVape, steepDays, flavors, percentVG } = this.props;
-    const percentPG = 100 - percentVG;
-    const flavorTotal = flavors.reduce((accum, current) => {
-      return accum + current.percent;
-    }, 0);
-
-    return (
-      <Table striped bordered>
-        <caption className="text-center">
-          {maxVG ? (
-            <span>Max VG |&nbsp;</span>
-          ) : (
-            <span>
-              {percentPG}% PG / {percentVG}% VG |&nbsp;
-            </span>
-          )}
-          {shakeAndVape ? (
-            <span>Shake &amp; Vape |&nbsp;</span>
-          ) : (
-            <span>Steep for {steepDays} days |&nbsp;</span>
-          )}
-          <span>Flavor total: {flavorTotal}%</span>
-        </caption>
-        <thead>
-          <tr>
-            <th>Flavor</th>
-            <th>Percent</th>
-            <th>Stash</th>
+  return (
+    <Table striped bordered>
+      <caption className="text-center">
+        {maxVg ? (
+          <span>Max VG |&nbsp;</span>
+        ) : (
+          <span>
+            {percentPg}% PG / {percentVg}% VG |&nbsp;
+          </span>
+        )}
+        {shakeAndVape ? (
+          <span>Shake &amp; Vape |&nbsp;</span>
+        ) : (
+          <span>Steep for {steepDays} days |&nbsp;</span>
```

Comment:
I know this isn't from your PR but just as a general note we should figure out a better way to space this out than using `&nbsp;`